docs/setup.md

```diff
@@ -19,7 +19,7 @@ Make sure you have one or the other or both:
 
 ### Environment File
 
-Populate a `.env` file by using the examples from [`.env.example`](../.env.example). The `.env` file should **never** be committed.
+Populate `.env` and `cypress-ete.env` files by using the examples from [`.env.example`](../.env.example) and [`cypress-ete.env.example`](../cypress-ete.env.example.example). These `.env` files should **never** be committed as they store secrets.
```

Comment:
the `.env` file was being used for both local development and running ete tests locally (we have a separate env file for mocked tests). Extracting the ete tests into their own env file makes it easier to test for the presence or absence of ccokies set on the server response.

Reply:
Is it worth uploading a `cypress-ete.env` file to s3 too, and a link in the documentation to download it?